### DISCUSSION TITLE
Adventure map, position the view of the adventure map automatically to match the map of the "View Puzzle" screen

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#10295] [General]
In fheroes2, it is sometimes difficult to position the adventure map precisely in relation to the puzzle with the arrow shaped mouse cursor with the edges of the screen for example.
This was also somewhat the case in the original game, but to a lesser extent.

To overcome this problem, I suggest that when a part of the displayed adventure map is present in an already revealed part of the "View Puzzle" screen when the player exits the "View Puzzle" screen the adventure map is automatically positioned to exactly match the map of the "View Puzzle" screen.
When the player exits the "View Puzzle" screen by pressing the "EXIT" button a sound is played to indicate to the player that the view of the adventure map has been adjusted to exactly match the map on the "View Puzzle" screen.

This may eventually be an option for those who find it too easy this way. :)

**Comment by ihhub** [comment]
This isn't a feature of any game in the series. It is a responsibility of a player to find out the precise position of the Ultimate Artifact.
